### DISCUSSION TITLE
ios 传输照片到电脑建议增加全选功能

### DISCUSSION THREAD

**Discussion by xuefliang** [localsend/localsend#1273] [L-简体中文]
现在版本没有全选功能，100多个照片手动选择太费劲

**Comment by catnuko** [comment]
是的，我备份1000个视频，一个个选中的

**Comment by summerxiaohao** [comment]
可以在手机相册里面滑动选中，点击左下角发送，选择local send，算是个曲线方法把，跳过自带那个一个个点的选择器，用iOS相册选择器选择……

**Reply by catnuko** [reply]
那个选的文件太多太大时会很卡很久才开始发送

**Reply by Zelayan** [reply]
相册的选择不是包含所有数据的

**Reply by chunjoe** [reply]
這樣會把HEIC的照片轉成JPG...傻眼
---
更新，再選項裡面可以設定"目前"格式

**Comment by steenk00** [comment]
计划发送1200个图片的我，搜索到了这个需求，同求

**Comment by tableking** [comment]
同样的需求，增加全选功能真的很重要。

**Comment by weiweizhen** [comment]
哇，这个功能顶顶顶

**Comment by 2han9wen71an** [comment]
不知道何时能上，手动选了300多张后不想用了
